payloads/library/recon/InfoGrabber/info.ps1

```diff
@@ -113,29 +113,7 @@ $drivers=Get-WmiObject Win32_PnPSignedDriver| where { $_.DeviceName -notlike $nu
 # videocard
 $videocard=Get-WmiObject Win32_VideoController | Format-Table Name, VideoProcessor, DriverVersion, CurrentHorizontalResolution, CurrentVerticalResolution
 
-#Get installed passwords

```

Comment:
I'm not too familiar with windows, but I'm curious why this was removed. Is it redundant?

Reply:
Yes it was. It was another method of getting stored WiFi passwords. It was less reliable though as it relied on splitting at the ':'. This was a problem if an ESSID contained a ':', as it would split incorrectly and cause errors.
The other method called near the top of the script works better and is more robust overall, working with ESSIDs that have a ':' in them by splitting a slightly different way.
Also, even though it was run, it's output was never used and just caused some junk output (a few numbers that increment) to appear at the top of the output log.
Thus, I removed it speeding up the process, and eliminating the junk in the log.

Reply:
Thanks for the explanation!

Reply:
You're welcome! Thanks for the merge!